sklearn/tests/test_common.py

```diff
@@ -296,6 +297,54 @@ def test_estimators_overwrite_params():
                 yield check_estimators_overwrite_params, name, Estimator
 
 
+# These are the known trailing-_ properties on current estimators.
```

Comment:
Are any of the current ``_KNOWN_PROPERTIES`` instance attributes or are all class attributes? If they are class attributes, we could easily add that as an exception, right?